src/js/main.js

```diff
@@ -18,6 +18,12 @@ export function init(el, context, config) {
         const youTubeId = resp.sheets[config.sheetName][0].youTubeId;
         const chapters = resp.sheets[config.sheetChapter];
 
+        chapters.forEach( function(chapter, index){
+          if(chapters.length > index+1){
+            var endChapter = chapters[index+1];
+            chapter.endChapter = endChapter.chapterTimestamp -1;
```

Comment:
I think `endChapter.chapterTimestamp` is a string at this point, so we'd need to `parseInt(endChapter.chapterTimestamp)` before `-1` for safety